2-mandatory/1-weather-report.js

```diff
@@ -12,10 +12,14 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    let newArray = [];
+    for (let i = 0; i < cities.length; i++){
+        newArray.push("The temperature in " + cities[i] + " is " + temperatureService(cities[i]) + " degrees");
+    
+    }
+    return newArray;
 }
 
-
 /* ======= TESTS - DO NOT MODIFY ===== */
 
 function temperatureService(city) {
```

Comment:
Try and get into the habit of using a more descriptive name for your variable than 'newArray'